### ISSUE TITLE
why i cannot get a good resuilt?

### ISSUE THREAD

**Issue by maxiaoming6669** [qijiezhao/M2Det#92]
I trained 15,000 data and applied the pre-training model. The lowest LOSS returned during training was only 0.5. None of the test results were detected, even in the test training set. I use VOC-type datasets

**Comment by williamluke4** [comment]
Yeah, I'm having the same Issue, I trained on COCO 2014. 
![image](https://user-images.githubusercontent.com/10773719/63240262-8c1a7c00-c257-11e9-9ef8-26ca3887dba7.png)

**Comment by maxiaoming6669** [comment]
Who can tell me why?

**Comment by hshsjkjy** [comment]
Has anyone trained a good result? The two loss values have been around 1 and the detection is very poor.

**Comment by kuwt** [comment]
> Yeah, I'm having the same Issue, I trained on COCO 2014.
> ![image](https://user-images.githubusercontent.com/10773719/63240262-8c1a7c00-c257-11e9-9ef8-26ca3887dba7.png)

I got the same result.
![image](https://user-images.githubusercontent.com/20258451/64834821-216e1d80-d616-11e9-9bdc-4e44d6c8a01d.png)

**Comment by nile649** [comment]
Same here

**Comment by williamluke4** [comment]
This may shed some light https://github.com/qijiezhao/M2Det/issues/81#issuecomment-510857633
**Translated**
"The effect of training is not good, of course, it must be solved from training. Adjusting the hyperparameters such as lr batch during training and training the model well, this problem is naturally solved. If the training effect is not good, or depends on how your current specific hyperparameters are set, you can record the training loss and test accuracy during training, and then adjust the hyperparameter according to the loss curve accuracy curve."

**Comment by shuoshuoa** [comment]
i also has bad result in KITTI.

**Comment by zhangxiang1209** [comment]
I get the same issue, why??? how to solve it???
![image](https://user-images.githubusercontent.com/45326776/66916704-612d8800-f04e-11e9-9b21-031b8aec4284.png)
